kymatio/frontend/torch_f.py

```diff
@@ -0,0 +1,23 @@
+import torch.nn as nn
+
+class Scattering(nn.Module):
+   def __init__(self, J, shape, max_order = None):
+       super(Scattering, self).__init__()
+       return
```

Comment:
Why this statement?

Also please explain the naming of the file

Reply:
So - I wanted to do a "template" for a recurrent type of class as it's done in standard OOP. Maybe it's a bad idea here... any suggestions?

Reply:
fixed